### ISSUE TITLE
Contributor Images missing ALT Text

### ISSUE THREAD

**Issue by bbrazis** [michaella23/bootcamp-collab#113]
This is an accessibility issue.

My thoughts are that if someone has added included a link to their profile image it should add `alt="${name}'s profile picture"` and if it is empty the alt text should be something like `alt=""` that way it doesn't flag any accessibility warnings.

**Comment by michaella23** [comment]
Thanks for creating this issue, @bbrazis. I wonder the best way to handle this. We could add an alt text property in each 'contributor' object maybe. Do you think we would add that attribute inside this function? (This starts at line 42 in [`contributors.js`](https://github.com/michaella23/bootcamp-collab/assets/javascript/contributors))

```const createUserImgEl = function (index) {
  const userImgEl = document.createElement('img');
  const ifUserImage = contributors[index].profileImage ? contributors[index].profileImage : randomDefaultProfilePic();
  userImgEl.classList.add('scrimba-contributors--user-img');
  userImgEl.setAttribute('src', ifUserImage);

  return userImgEl;
}```

**Comment by bbrazis** [comment]
Probably something like:
`if(contributors[index].profileImage){
const name = contributors[index].name;
userImgEl.setAttribute('alt', `${name}'s profile picture`);
} else {
userImgEl.setAttribute('alt', '');
};
`

**Comment by michaella23** [comment]
If you want to do some work on this issue, go for it @bbrazis.  If we need other contributors to add info as well, we can let them know.

**Comment by bbrazis** [comment]
Went ahead and made a pull request for this. Hopefully it doesn't destroy anything  🙃

**Comment by michaella23** [comment]
> Went ahead and made a pull request for this. Hopefully it doesn't destroy anything 🙃

Might be fun to destroy something and see how to fix it! 😂 (for future reference...)
